.github/workflows/add-geometry-reviewers.yaml

```diff
@@ -0,0 +1,18 @@
+# This workflow adds geometry reviewers to PRs labled with Geometry
+
+name: add-geometry-reviewers
+
+on:
+  pull_request:
+    types: [ labeled ]
+
+jobs:
+  build:
+    if: ${{ github.event.label.name == 'geometry' }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: Add Reviewers
+      uses: madrapps/add-reviewers@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        reviewers: dassaf4,saeeedtorabi
```

Comment:
you might want to check the api, don't know if there's a different step for adding assignee vs adding a review, but you could use a condition like if `github.event_name == 'pull_request'` add reviewer, `if github.event_name == 'issue' add assignee.